### ISSUE TITLE
Improving storage of graph metadata

### ISSUE THREAD

**Issue by nomisto** [OpenBioLink/OpenBioLink#66]
Currently the format for storing metadata of the graph (database, infiles, edges, ...) in the graph creation module is a little confusing. Maybe looking into storing metadata of the sources, edges, etc. in a more centralized format (f.e. json) could improve readability. Further as sources can change at anytime (seperator, urls, ...) this would help in adatpting to these changes faster and would make  including new sources/edges easier.

**Comment by matthias-samwald** [comment]
Do you think we could standardize everything to RDF?

**Comment by nomisto** [comment]
Yes that might be even better than JSON. With RDF we could provide more readability than the current method (storing metadata in seperate static python classes), while still having the modularity of the current method.

**Comment by matthias-samwald** [comment]
I guess than we should try to transition to RDF! 
One potential problem is of course that sometimes the triples make it a bit cumbersome to capture N-ary relationship with N>3. But that is probably not a problem for the dataset descriptions.
